### DISCUSSION TITLE
Prisma migrate with unsupported index type

### DISCUSSION THREAD

**Discussion by shulmaneyal** [prisma/prisma#11496] [Q&A]
I have a model for which I am using an unsupported index type (mssql columnstore). As instructed in the docs, I am creating a migration using the --save-only flag and edit the generated sql code to include the index, this works as expected. But then every time I run a new migration without changing the model it drops the manually created index, is there a way to avoid it?

Manually added sql statement for the first migration:
```
CREATE CLUSTERED COLUMNSTORE INDEX [CCI_journal] ON [dbo].[Journal];
```

New migration drop index statement:
```
DROP INDEX [CCI_journal] ON [dbo].[Journal];
```

**Comment by janpio** [comment]
That might unfortunately be how this works. Can you link to the docs that explain how to do this? We might need to re-investigate and possibly add this disclaimer.

**Reply by vladi-stevanovic** [reply]
👋 Hello @shulmaneyal I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! 🙌

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
